stl/inc/memory

```diff
@@ -1252,10 +1253,47 @@ private:
     }
 
     template <class _Ty0, class... _Types>
-    friend shared_ptr<_Ty0> make_shared(_Types&&... _Args);
+#if _HAS_CXX20 && defined(__cpp_concepts)
+    requires(!is_array_v<_Ty0>)
+#endif // _HAS_CXX20 && defined(__cpp_concepts)
+        friend shared_ptr<_Ty0> make_shared(_Types&&... _Args);
+
+#if _HAS_CXX20 && defined(__cpp_concepts)
+    template <class _Ty0>
+    requires is_unbounded_array_v<_Ty0> friend shared_ptr<_Ty0> make_shared(size_t _Count);

```

Comment:
clang-format appears to be completely brocken with respect to `requires` clauses

Reply:
Fixed it by disabling clang-format in these regions of code